spec.md

```diff
@@ -564,9 +564,25 @@ the resource it creates.
 | service_id* | string | ID of the service from the catalog. MUST be a non-empty string. |
 | plan_id* | string | ID of the plan from the catalog. MUST be a non-empty string. |
 | app_guid | string | Deprecated in favor of `bind_resource.app_guid`. GUID of an application associated with the binding to be created. If present, MUST be a non-empty string. |
-| bind_resource | JSON object | A JSON object that contains data for platform resources associated with the binding to be created. Current valid values include `app_guid` for [credentials](#types-of-binding) and `route` for [route services](#route-services). |
+| bind_resource | JSON object | A JSON object that contains data for platform resources associated with the binding to be created. See [Bind Resource Object](#bind-resource-object) for more information. |
 | parameters | JSON object | Configuration options for the service binding. An opaque object, controller treats this as a blob. Brokers SHOULD ensure that the client has provided valid configuration parameters and values for the operation. |
 
+##### Bind Resource Object
+
+The `bind_resource` object contains platform specific information
```

Comment:
I prefer formatting of documentation source to be as close to the user-consumed product as possible. As I wouldn't publish documentation with sentences arbitrarily wrapped like a poem, and which can't expand with browser width, and I can't assume when new lines in source are honored or joined, I prefer them joined in source.